### PR TITLE
#70-Night mode

### LINKED ISSUE #70: Night Sky Map function
Add the night sky map for the user to view different version of maps.

### PR DESCRIPTION
**Pull request by munozirianni1988**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #70

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Comment by munozirianni1988** [comment]
Screenshot from 2023-10-19 23-15-08

File: client/src/components/MapSection.css [hunk 1]
```diff
@@ -21,10 +19,45 @@
...
+}
+
+.bootstrap-container span {
+	margin-top: 10px;
+}
+
+.bootstrap-light {
+	background-color: white;
+	box-shadow: 10px 10px 12px -8px rgba(0, 0, 0, 0.75);
+	color: #cb3100;
```
Comment (mferryRV) [inline client/src/components/MapSection.css:54]: Add a border or different background color in light mode to distinguish that it's a button

File: client/src/components/MapSection.js [hunk 2]
```diff
@@ -1,26 +1,43 @@
...
 function MapSection() {
 	const [isFullScreen, setFullScreen] = useState(false);
+	const [isNightMode, setNightMode] = useState(false);
 
 	const toggleFullScreen = () => {
 		setFullScreen(!isFullScreen);
 	};
 
+	const toggleNightMode = () => {
+		console.log("night mode");
```
Comment (mferryRV) [inline client/src/components/MapSection.js]: No console logs in production

File: client/src/components/MapSection.js [hunk 3]
```diff
@@ -1,26 +1,43 @@
...
+			}`}
+		>
+			<div className="bootstrap-container">
+				<span>Click on locations to learn more about local rhythms</span>
+				<Button
+					variant={isNightMode ? "dark" : "light"}
+					onClick={toggleNightMode}
+					className={isNightMode ? "bootstrap-night" : "bootstrap-light"}
+				>
+					{isNightMode ? "Light Map" : "Night Map"}
```
Comment (mferryRV) [inline client/src/components/MapSection.js]: Light Mode / Dark Mode

**Review by mferryRV — APPROVED** [review]
Minor changes - don't worry about CSS alignment as we'll fix that with the mobile view

**Pull request merged**